packages/manager/modules/enterprise-cloud-database/src/card/card.scss

```diff
@@ -0,0 +1,81 @@
+
+$default-background-color: #fff;
+$default-border-color: #d2d1d1;
+$selected-background-color: rgba(129, 211, 248, 0.1);
+$selected-border-color: rgba(2, 167, 240, 1);
+
+.card {
+    border-radius: 3px;
+    border: 1px solid $default-border-color;
+    box-sizing: border-box;
+    height: 100%;
+    cursor: pointer;
+    background-color: $default-background-color;
+}
+
+.card .card-body {
```

Comment:
Should be nested in `.card`

Reply:
we have a plan to move this component to UI KIT. Will take care of these during the migration.